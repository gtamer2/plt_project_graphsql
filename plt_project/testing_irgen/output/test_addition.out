; ModuleID = 'GraphSQL'
source_filename = "GraphSQL"

define i32 @main() {
entry:
  ret i32 0
}
