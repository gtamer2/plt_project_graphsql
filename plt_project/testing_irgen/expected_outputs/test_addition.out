Processing all stmts
Initial Expression: (5 + 6) 
Evaluating expression: (5 + 6)
Evaluating expression: 5
Evaluating expression: 6
Result: true
